package-lock.json

```diff
@@ -70,7 +74,7 @@
         "react-native-simple-markdown": "^1.1.0",
         "react-native-svg": "12.1.1",
         "react-native-swipe-gestures": "^1.0.5",
-        "react-native-tuvali": "github:mosip/tuvali#v0.4.1",
+        "react-native-tuvali": "file:.yalc/react-native-tuvali",
```

Comment:
please refrain from adding the yalc add in the repo. yalc is mainly used for local testing. please mention the exact tuvali git hub tag version.

Reply:
I reverted the exact Tuvali version after removing Yalc in package-lock.json